### PR TITLE
TEST-#3340: add groupby multicolumn case and '.agg' case

### LINKED ISSUE #3340: [ASV] add groupby multicolumn case for omnisci backend
An important case for that we want to track performance.

### PR DESCRIPTION
**Pull request by anmyachev**
Signed-off-by: Anatoly Myachev <anatoly.myachev@intel.com>

## What do these changes do?

- [ ] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/contributing.html)
- [ ] passes `flake8 modin`
- [ ] passes `black --check modin`
- [ ] signed commit with `git commit -s` 
- [x] Resolves #3340 
- [ ] tests added and passing
- [ ] module layout described at `docs/developer/architecture.rst` is up-to-date

### CODE REVIEW

**Comment by anmyachev** [comment]
Blocked by https://github.com/modin-project/modin/issues/3197

File: asv_bench/benchmarks/omnisci/benchmarks.py
Comment (dchigarev) [inline asv_bench/benchmarks/omnisci/benchmarks.py:435]: please also add `groupby_sum` to the `MultiColumn` benchmark, so we can see performance differences with its equivalent in a single-column groupby. 

"mean" and "nunique" will default to pandas for some time after #3250 will be merged, so their time would be kinda irrelevant to track the performance of multi-column groupby

Comment (anmyachev) [reply asv_bench/benchmarks/omnisci/benchmarks.py:438]: `groupby_sum` case is added

**Review by YarShev — APPROVED** [review]
LGTM

**Comment by YarShev** [comment]
@anmyachev , ping me when CI passes.

**Comment by anmyachev** [comment]
@YarShev we can merge now

**Pull request merged**
